module/Photo/src/Photo/Mapper/Album.php

```diff
@@ -151,6 +151,61 @@ public function getRootAlbums()
     }
 
     /**
+     * Gets all root albums containing photos made between the specified dates
```

Comment:
Commentaar klopt niet met de de functie. De query geeft alle albums die hun eerste foto tussen de data heeft. Als een album eerder begonnen is, maar zijn laatste foto na $start heeft, dan word hij niet gevonden